layouts/shortcodes/pages/compatibility-products.html

```diff
@@ -1,18 +1,9 @@
 <div class="row">
-  <div class="col-md-16">{{ i18n "compatibility-intro-text" | safeHTML }}</div>
-  <div class="col-md-8"><img class="compatible-logo img img-responsive" src="/images/jakarta/jakarta-ee-compatible-logo-color.svg" title="{{ i18n "jakarta-ee-logo-alt" }}"></div>
+  <div class="{{ with .Page.Params.compatibility_download_wrapper }}{{ . }}{{ end }}" style="text-align: center;">
+    {{ i18n "compatibility-download-button" | safeHTML }}
+    <a class="btn btn-lg btn-secondary margin-bottom-5" href="https://githubcom/eclipse-ee4j/jakartaee-platform/issues?q=is%3Aissue+is%3Aclosed+label%3Acertification">Compatibility Requests</a></div>
```

Comment:
I'm doing this `compatibility_download_wrapper` because I added Chinese "compatibility-download-button", and in Chinese, the button is shorter, I need to keep the button align right for both Zh and En pages.